### ISSUE TITLE
You get exactly one chance to save the QR code!

### ISSUE THREAD

**Issue by MaffooClock** [JaneJeon/blink#817]
## Disclaimers

- [x] Before filing this report, I have read the documentation fully and followed it.
- [x] I understand that if the "issue" is something that is already answered by the documentation and/or are caused by me not reading the documentation, the issue _will_ be blocked and/or locked (with the implicit explanation being to "go read the docs"), and I may not be able to open another issue for this repository ever again.

## Issue Description

**Describe the bug**
When creating a link, the QR code is shown only once: in the modal after a successful save; there's no way to ever see it again.

**Deployment Method**
Production:
- AWS ECS Fargate (behind Application Load Balancer) + AWS RDS (PostgreSQL 16.3) + AWS ElastiCache (Redis 6.2.6)

Local testing: 
- Docker container from [deploy/docker-compose/docker-compose.yml](https://github.com/JaneJeon/blink/blob/8619fa7fa69e679cad1fdf475476e5bbb44512b0/deploy/docker-compose/docker-compose.yml)

**To Reproduce**
1. Create a new link, entering any valid URL
2. Press <kbd>Enter</kbd> (no save button?)
3. QR code will be displayed -- the one and only time, ever

**Expected behavior**
There should be a way to display the QR code to the user for any existing link

**Screenshots**
N/A

**Additional context**
Issue #572

**Comment by sistason** [comment]
You can just click on the link in the list of links (`/app/links`), where you also edit the link. On the bottom, there is the QR Code
